examples/topology.example.yaml

```diff
@@ -211,6 +211,54 @@ tiflash_servers:
 #   - host: 10.0.1.21
 #   - host: 10.0.1.22
 
+# NOTE: Only 1 master node is supported for now
+tispark_masters:
+  - host: 10.0.1.21
```

Comment:
is it possible to have multiple spark masters?

Reply:
Is there any way to setup multiple masters without setting up a zookeeper? I didn't see such configs in `tidb-ansible`.

Reply:
I think we can remain the logic to support only one master node as it is.